summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -162,13 +162,7 @@ abstract class Storm(options: Map[String, Options], transformConfig: Summingbird
   def withConfigUpdater(fn: SummingbirdConfig => SummingbirdConfig): Storm
 
   def plan[T](tail: TailProducer[Storm, T]): PlannedTopology = {
-    /*
-     * TODO: storm does not yet know about ValueFlatMapped, so remove it before
-     * planning
-     */
-    val dagOptimizer = new DagOptimizer[Storm] {}
-    val stormTail = dagOptimizer.optimize(tail, dagOptimizer.ValueFlatMapToFlatMap)
```

Comment:
We have support on the level we can build `flatMap` operation over it (https://github.com/twitter/summingbird/commit/44fb5c9#diff-1d4313fb460f1921fc349a8eb2ee902fR85). But it's not fully supported in a way to affect `OnlinePlan`. I would like to do this in future, but I guess it's out of this PR. It's about when do not trigger new node creation, and this one is when to trigger it =)